src/test/safe.dom.upgrade.test.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import Web3 from 'web3'
+import GnosisSafeSol from '@gnosis.pm/safe-contracts/build/contracts/GnosisSafe.json'
+import {
+  DEFAULT_FALLBACK_HANDLER_ADDRESS,
+  getEncodedMultiSendCallData,
+  SAFE_MASTER_COPY_ADDRESS,
+} from '~/logic/contracts/safeContracts'
+import { ZERO_ADDRESS } from '~/logic/wallets/ethAddresses'
+
+
+describe('Upgrade a Safe', () => {
+  it('Calls getEncodedMultiSendCallData with a list of MultiSendTransactionInstanceType and returns the multiSend data encoded', async () => {
```

Comment:
what do you test here? the safe contract? I don't understand the value this test brings to the interface

Reply:
Tests that the method `getEncodedMultiSendCallData` encodes the multisend transaction data ok